dashboard/config/scripts/levels/allthethings design mode elements.level

```diff
@@ -167,4 +167,4 @@
   }
 }]]></config>
   <blocks/>
-</Applab>
\ No newline at end of file
+</Applab>
```

Comment:
It's possible we'd get a merge conflict here if someone modified the same level via levelbuilder. If that happens I can "accept theirs" and redo my changes to this level. I suppose in this case it would have been safe to simply modify the level on levelbuilder first, since I'm not changing the behavior of this level.